README.md

```diff
@@ -1,39 +1,38 @@
 # weave-kube
-Weave Net integration with Kubernetes for seamless turn-up
 
-This repo contains a sample DaemonSet config for Kubernetes to install
+Weave Net integration with Kubernetes for seamless turn-up.
+
+This repo contains a Kubernetes add-on (a DaemonSet) to install
 Weave Net with one command, and the source code for the image which
 makes that happen.
 
 ## Prerequisites:
 
-A version of Kubernetes that includes PR #28178, to reload the CNI
-configuration after launch. (So something newer than the 1.3.x releases)
-
-Download a copy of `weave-daemonset.yaml` from this repo.
+Kubernetes 1.3 or newer.
```

Comment:
Why 1.3?  If you don't have that change I mentioned, it won't re-read the CNI config.  Maybe you can get past that by restarting kubelet on every node, but we'd have to say that.